rn/Teacher/src/routing/Navigator.js

```diff
@@ -30,6 +30,7 @@ type ShowOptions = {
   modal: boolean,
   modalPresentationStyle: string,
   embedInNavigationController: boolean,
+  allowSwipeDownToDismissModal: boolean,
```

Comment:
My comment showed up in a weird place. I was referring to this. Did you mean `disableSwipeDownToDismissModal`?

Reply:
doh!  fixed.